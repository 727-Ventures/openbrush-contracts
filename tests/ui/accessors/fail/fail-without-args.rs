#[openbrush::storage_item]
#[openbrush::accessors]
#[derive(Default, Debug)]
pub struct AccessData {
    #[get]
    #[set]
    read_write: u32,
    #[get]
    read_only: u32,
    #[set]
    write_only: u32,
}

fn main() {}