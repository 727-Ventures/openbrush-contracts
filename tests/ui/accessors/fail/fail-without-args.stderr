error: expected identifier, found `{`
 --> tests/ui/accessors/fail/fail-without-args.rs:6:1
  |
6 | #[openbrush::accessors]
  | ^^^^^^^^^^^^^^^^^^^^^^^ expected identifier
  |
  = note: this error originates in the attribute macro `openbrush::accessors` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0412]: cannot find type `AccessData` in this scope
 --> tests/ui/accessors/fail/fail-without-args.rs:8:12
  |
8 | pub struct AccessData {
  |            ^^^^^^^^^^ not found in this scope

error[E0422]: cannot find struct, variant or union type `AccessData` in this scope
 --> tests/ui/accessors/fail/fail-without-args.rs:8:12
  |
8 | pub struct AccessData {
  |            ^^^^^^^^^^ not found in this scope

warning: unused import: `openbrush::traits::Storage`
 --> tests/ui/accessors/fail/fail-without-args.rs:3:5
  |
3 | use openbrush::traits::Storage;
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default
